src/App.scss

```diff
@@ -157,6 +157,12 @@ header h1 {
 	top: 15px;
 }
 
+// Adjust color of slider arrows
+.awssld__controls{
+  --organic-arrow-color: black !important;
```

Comment:
I somehow like the black contrast here, but probably you could see if you like a light grey with a thin black border more (or without border but with a tiny shadow)

Reply:
I tried a bit, but I can't figure out how to access border-property... this comes from awesome slider module. And in dev tools nothing happens when adding border property or --organic-arrow-border property.